chain/chain/src/test_utils.rs

```diff
@@ -27,14 +28,14 @@ use near_store::{Store, StoreUpdate, Trie, TrieChanges, WrappedTrieChanges, COL_
 use crate::error::{Error, ErrorKind};
 use crate::store::ChainStoreAccess;
 use crate::types::{
-    ApplyTransactionResult, BlockHeader, RuntimeAdapter, StatePart, StatePartKey,
+    ApplyTransactionResult, BlockHeader, RuntimeAdapter, StatePartKey,
     ValidatorSignatureVerificationResult, Weight,
 };
 use crate::{Chain, ChainGenesis, ValidTransaction};
 use near_primitives::errors::RuntimeError;
 use near_primitives::merkle::{merklize, verify_path, MerklePath};
 
-pub const DEFAULT_STATE_NUM_PARTS: u64 = 17; /* TODO MOO */
+pub const DEFAULT_STATE_NUM_PARTS: u64 = 17;
```

Comment:
Looks like a magic number. Should we keep a TODO to remove it by placing in some config?

Reply:
This constant is only in test_utils and never used somewhere else. My first thought, no one will care about its value in future (except testing). The second one, the whole logic may be modified when tests with huge state will appear.

For me TODO is fine, magic number is fine and config works too. I will add a comment there and we'll see how it helps.

Reply:
@nearmax does it work?

Reply:
> The second one, the whole logic may be modified when tests with huge state will appear.

Sounds good.